magento/pwa-studio
# [PB] Carousel appearance for Product content type

## Description
Add support of carousel appearance in Products component

## Related Issue
[https://jira.corp.magento.com/browse/PB-175](https://jira.corp.magento.com/browse/PB-175)

## Verification Steps
1. Check that each property for each content type appearance renders correctly on PWA Venia storefront in desktop and mobile.
2. Compare Venia PWA rendering of content type with Luma to ensure the experience is the same or better for each property of each appearance in desktop and mobile.
3. Check each supported browser and on mobile that is supported according to PWA documentation.
4. Acceptable performance has been validated

Notice: Do not merge until ticket will be in status `Acceptance - Done`
